packages/embedded-wallet-service-wagmi/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
+    "esModuleInterop": true
```

Comment:
I don't think we need lines 2 - 4. This is already specified in the `js-library.json` which itself inherits from the `base.json`